django/db/models/aggregates.py

```diff
@@ -145,6 +145,19 @@ class Sum(FixDurationInputMixin, Aggregate):
     name = 'Sum'
     allow_distinct = True
 
+    def __init__(self, *expressions, default=None, **extra):
+        self.default_value = default
+        super().__init__(*expressions, **extra)
+
+    def convert_value(self, value, expression, connection):
+        value = self.default_value if value is None else value
+
+        import datetime
+        if isinstance(value, datetime.timedelta):
+            value = value.microseconds
```

Comment:
I don't understand why this is here? Also why not `value.total_seconds()`? Or am I missing something?

```python
>>> from datetime import timedelta
>>> delta = timedelta(microseconds=90000000123)
>>> delta
datetime.timedelta(days=1, seconds=3600, microseconds=123)
>>> delta.microseconds
123
>>> delta.total_seconds()
90000.000123
```